src/strings/normalize.c

```diff
@@ -561,9 +561,20 @@ static MVMint32 should_break(MVMThreadContext *tc, MVMCodepoint a, MVMCodepoint
                 return 0;
             if ( GCB_b == MVM_UNICODE_PVALUE_GCB_ZWJ )
                 return 0;
+            if ( GCB_b == MVM_UNICODE_PVALUE_GCB_GLUE_AFTER_ZWJ )
```

Comment:
I wonder a little if a `switch` would be faster here, but maybe the compiler will figure that out anyway...